### ISSUE TITLE
UX for archived token balances

### ISSUE THREAD

**Issue by piyalbasu** [stellar/freighter#1438]
```[tasklist]
### Tasks
```

**Comment by conveyGhost** [comment]
hello. @piyalbasu 
What can I help you for UX for archived token balances?

**Comment by sdfcharles** [comment]
Context: https://stellarfoundation.slack.com/archives/C016CPSLVLG/p1726523080699389

**Comment by sdfcharles** [comment]
Designs: https://www.figma.com/design/C3G0a4Gd6RQyplRBppGDsL/Freighter-(SDS-v3)?node-id=2932-3383&t=KhdFFpbyyxRgbwuo-1

**Comment by JakeUrban** [comment]
@sdfcharles I only see designs for extending TTLs, and we have a separate task for that (#1689). What is the UX for when token balances have been archived?

**Comment by sdfcharles** [comment]
Updated designs here: https://www.figma.com/design/C3G0a4Gd6RQyplRBppGDsL/Freighter-(SDS-v3)?node-id=2932-3383&t=NNZIouOPrlmRWQS4-1

Included states for when a token balance is archived and a notification on the home screen when a token balance has been archived

**Comment by JakeUrban** [comment]
Thanks @sdfcharles, but I'm not sure if we want to even inform a user that their balance is archived, because they actually won't have to explicitly restore their balances before they can use them after protocol 23 is live. 

After protocol 23, if a transaction involves an archived entry, it will automatically be restored when the transaction is executed, and all the user needs to do is make sure their fee is high enough to pay for the restoration.

What that means for us is that regardless of whether a transaction is sent to freighter from a dapp or a transaction is initiated from within freighter, we can go about our normal signing and submission flow without having to do anything different in our UX for transactions that use archived entires.

**Comment by aristidesstaffieri** [comment]
> Thanks [@sdfcharles](https://github.com/sdfcharles), but I'm not sure if we want to even inform a user that their balance is archived, because they actually won't have to explicitly restore their balances before they can use them after protocol 23 is live.
> 
> After protocol 23, if a transaction involves an archived entry, it will automatically be restored when the transaction is executed, and all the user needs to do is make sure their fee is high enough to pay for the restoration.
> 
> What that means for us is that regardless of whether a transaction is sent to freighter from a dapp or a transaction is initiated from within freighter, we can go about our normal signing and submission flow without having to do anything different in our UX for transactions that use archived entires.

Do you think that it makes sense to inform the user of why their fee is higher than without the accompanying restore operation, or will the fee difference be negligible?

 Maybe only more sophisticated users will even notice this but I feel it could be weird to prepare a tx that you might do often and have it sometimes have a higher fee than other times.

**Comment by JakeUrban** [comment]
Yea I would think ideally Freighter would give you detailed info on why your fee is what it is, but I don't know if we can do that for all transactions yet. I don't see any details on cost provided in RPC's [simulation endpoint](https://developers.stellar.org/docs/data/rpc/api-reference/methods/simulateTransaction) response definition. Maybe this something already discussed on the platform team? cc @mollykarcher 

So maybe for now we can do something specifically for when transactions restore archived entries. As for what level of detail to provide, I'm not sure. I would think just some UX acknowledgment that the fee is higher due to restoring necessary entries would be sufficient, as opposed to actually providing the entires that are being restored as a result of executing the transaction.

**Comment by aristidesstaffieri** [comment]
> Yea I would think ideally Freighter would give you detailed info on why your fee is what it is, but I don't know if we can do that for all transactions yet. I don't see any details on cost provided in RPC's [simulation endpoint](https://developers.stellar.org/docs/data/rpc/api-reference/methods/simulateTransaction) response definition. Maybe this something already discussed on the platform team? cc [@mollykarcher](https://github.com/mollykarcher)
> 
> So maybe for now we can do something specifically for when transactions restore archived entries. As for what level of detail to provide, I'm not sure. I would think just some UX acknowledgment that the fee is higher due to restoring necessary entries would be sufficient, as opposed to actually providing the entires that are being restored as a result of executing the transaction.

Yeah agree I think it makes sense to just at least have some sort of by-line or copy that informs you that "fee is higher than usual because you need to restore your entry before using it" but maybe in not so many words 😅

**Comment by JakeUrban** [comment]
Sounds like we've reached some consensus on how to handle state archival. @sdfcharles I'm putting this back into your backlog.
